includes/models/model.llms.user.certificate.php

```diff
@@ -111,4 +111,49 @@ public function get_user_postmeta() {
 		);
 	}
 
+	/**
+	 * Can user manage and make some actions on the certificate
+	 *
+	 * @since [version]
+	 *
+	 * @param int|null $user_id Optional. WP User ID (will use get_current_user_id() if none supplied). Default `null`.
+	 * @return bool
+	 */
+	public function can_user_manage( $user_id = null ) {
+
+		$user_id = $user_id ?? get_current_user_id();
+		$result  = ( $user_id == $this->get_user_id() || llms_can_user_bypass_restrictions( $user_id ) );
+
+		return apply_filters( 'llms_certificate_can_user_manage', $result, $this );
+
+	}
+
+	/**
+	 * Can user view the certificate
+	 *
+	 * @since [version]
+	 *
+	 * @param int|null $user_id Optional. WP User ID (will use get_current_user_id() if none supplied). Default `null`.
+	 * @return bool
+	 */
+	public function can_user_view( $user_id = null ) {
+
+		$user_id = $user_id ?? get_current_user_id();
+		$result  = $this->can_user_manage( $user_id ) || $this->is_sharing_allowed();
+
+		return apply_filters( 'llms_certificate_can_user_view', $result, $this );
```

Comment:
Add `$user_id` to parameters passed to filter and include docblock for the new filter.

```suggestion
		/**
		 * Filter whether or not a user can view a user's certificate.
		 *
		 * @since [version]
		 *
		 * @param boolean               $result      Whether or not the user can view the certificate.
		 * @param int                   $user_id     WP_User ID of the user viewing the certificate.
		 * @param LLMS_User_Certificate $certificate Certificate class instance.
		 **/
		return apply_filters( 'llms_certificate_can_user_view', $result, $user_id, $this );
```